### DISCUSSION TITLE
interactive items and containers

### DISCUSSION THREAD

**Discussion by stilnat** [RE-SS3D/SS3D#830] [Game Design]
If an item display in game, and if it can be picked up, should a left click on it always pick it up (or more generally, fire the primary interaction on the item) ?
Currently, lockers display items in them, but left clicking an item in a locker open the locker UI instead of picking up the item.
Is this behaviour wanted ?

**Comment by cosmiccoincidence** [comment]
I would think that picking up an item should have a higher priority.

**Reply by stilnat** [reply]
I agree but I thought you disagreed in cases such as the towel on the janicart. Or did you change your mind ?

**Reply by cosmiccoincidence** [reply]
Maybe I was confused. Happens to me a lot with how busy I am.
